src/main.rs

```diff
@@ -60,6 +59,11 @@ fn main() {
              args.flag_admin_host,
              args.flag_admin_port);
 
+    std::panic::set_hook(Box::new(|err| {
+        eprintln!("Panic error {:?}, exiting program.", err);
+        std::process::exit(77);
```

Comment:
why 77? is it some random exit code? :)

Reply:
Yeah, I didn't think of anything better, and it has to be not zero, so, I thought this random number :/